File: Código/Loja/src/main/java/banco/conexaoBanco.java
package banco;

import java.sql.Connection;
import java.sql.DriverManager;
import  java.sql.SQLException;

public class conexaoBanco {
    public static Connection createConnection() throws SQLException{
		String url = "jdbc:mysql://localhost:3306/loja"; //Nome da base de dados
		String user = "root"; //nome do usuário do MySQL
		String password = ""; //senha do MySQL

		Connection conexao = null;
		conexao = DriverManager.getConnection(url, user, password);

		return conexao;
	}
}

File: Código/Python/Loja/src/banco/ConexaoBanco.py
import mysql.connector

class ConexaoBanco:
    @staticmethod
    def get_connection():
        servidor = "localhost"
        porta = "3306"
        database = "loja"
        usuario = "root"
        senha = ""
        try:
            conexao = mysql.connector.connect(
                host=servidor,
                port=porta,
                database=database,
                user=usuario,
                password=senha
            )
            return conexao
        except mysql.connector.Error as e:
            print(f"Erro ao conectar: {e}")
            return None

if __name__ == "__main__":
    conexao = ConexaoBanco.get_connection()
    if conexao:
        print("Conexão estabelecida com sucesso!")
        conexao.close()